python/pyspark/pandas/series.py

```diff
@@ -944,6 +944,54 @@ def between(self, left: Any, right: Any, inclusive: bool = True) -> "Series":
 
         return lmask & rmask
 
+    def cov(self, other: "Series", min_periods: Optional[int] = None) -> float:
+        """
+        Compute covariance with Series, excluding missing values.
+
```

Comment:
Sorry, I forgot to mention, could you add `versionadded` with a version `3.3.0` here?

e.g., https://github.com/apache/spark/blob/master/python/pyspark/rdd.py#L2735

Reply:
thank you, updated